src/lib/components/Communities/CommunityHeader.js

```diff
@@ -0,0 +1,138 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2020-2021 CERN.
+// Copyright (C) 2020-2021 Northwestern University.
+// Copyright (C) 2021 Graz University of Technology.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from 'react';
+import { Container, Image, Icon, Placeholder } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+class CommunityHeaderIcon extends React.Component {
+  render() {
+    return (
+      <Icon
+        className="ml-5"
+        link
+        color="grey"
+        onClick={() => console.log('TODO: Open community modal')}
+        name="setting"
+      />
+    );
+  }
+}
+
+class CommunityHeaderLoading extends React.Component {
+  render() {
+    return (
+      <Container className="community-header">
+        <Placeholder>
+          <Placeholder.Header image>
+            <Placeholder.Line />
+            <Placeholder.Line />
+          </Placeholder.Header>
+        </Placeholder>
+      </Container>
+    );
+  }
+}
+
+class CommunityHeaderInfo extends React.Component {
+  render() {
+    const { community, imagePlaceholderLink } = this.props;
+    const logoLink = community ? community?.links?.logo : imagePlaceholderLink;
+    return (
+      <Container className="community-header">
+        <Image size="mini" src={logoLink} /> {community?.metadata?.title}
```

Comment:
what about missing logos ? I think @zzacharo had a solution for this

Reply:
I implemented a component here https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/Image.js

Reply:
is the issue created?